services/core-web/src/components/noticeOfWork/applications/permitGeneration/NOWPermitGeneration.js

```diff
@@ -75,8 +67,10 @@ export class NOWPermitGeneration extends Component {
 
   handlePremitGenSubmit = (values) => {
     if (this.props.isAmendment) {
+      // eslint-disable-next-line no-param-reassign
```

Comment:
Can you please add a comment as to why disabling these linting rules is needed?

Reply:
`no-param-reassign` is the linting rule being prevented from being triggered. Is that not enough?
Justin didn't have his linter running yesterday, so when I merged from develop today I had to quickly fix some issues.

Reply:
I've pushed up some code to fix the linting issue.